.github/workflows/cache-issues.yaml

```diff
@@ -0,0 +1,33 @@
+name: Cache issues status
+on:
+  schedule:
+    # Daily at 8:20 UTC
+    - cron: '20 8 * * *'
```

Comment:
pick other time, also consider running it more then once a day

Reply:
once per hour? IIRC limits are hour-based

Reply:
I think one a day should be enough

we don't want thing to start running too soon, 
consider a test currently running with version that doesn't yet have a fix.

we don't need this accurate to the hour level, one day is enough.